lib/dynflow/action/with_bulk_sub_plans.rb

```diff
@@ -0,0 +1,83 @@
+module Dynflow
+  module Action::WithBulkSubPlans
+    include Dynflow::Action::Cancellable
+
+    BATCH_SIZE = 10
```

Comment:
Let's rename this to `DEAFAULT_BATCH_SIZE`, as it can be changed by overriding `batch_size` method. Also, I would recommand making default a bit higher (even 100) should be safe for most cases, when not doing too much in the planning phase.